documentation/api/metrics/v1/mbeans.markdown

```diff
@@ -57,9 +57,9 @@ Responses return a JSON Object mapping strings to (strings/numbers/booleans).
 
 ### Database metrics
 
-* `com.jolbox.bonecp:type=BoneCP`: Database connection pool
-  metrics. How long it takes to get a free connection, average
-  execution times, number of free connections, etc.
+* See
```

Comment:
Yeah I'll update this section after I add in the metrics-registry integration, this is mostly filler for that PR.